templates/unminified/cache.tpl.html

```diff
@@ -0,0 +1,4 @@
+<!doctype html>
+<html manifest="manifest.appcache">
+<script defer async src="cache-iframe.js"></script>
+<title>c</title>
```

Comment:
Actually yes, otherwise it's not valid HTML5! (I didn't know about that before).